server.md

```diff
@@ -2,121 +2,6 @@
-
-Start by clicking on the green **clone or download** button in the upper right corner on [TabPy repository landing page](https://github.com/tableau/TabPy) and downloading the zip file. After unzipping, navigate to the folder containing `setup.bat` using Windows command line and type `setup.bat`. 
-
-The script carries out the following steps:
-
-  - Downloads and installs Anaconda, under the current user account for example `C:\users\yourUserName\` unless Anaconda is in the PATH or has a registry entry. Anaconda provides an exhaustive set of Python packages, including ML-related libraries that can be used by Python code executed by the server.
-  - Creates a Python environment named `Tableau-Python-Server` if one doesn't already exist. The script then activates the environment.
-  - Installs the required Python packages into the new environment, as well as the client package, as it contains common functionality that the server depends on.
-  - Initializes the server, displays the install location and instructions on how to start the server next time.
-
-After setup completes or when you run `startup.bat` per the instructions to start the server, you should see an output like this:
-
-```bash
-
-INFO:__main__:{"INFO": "Loading state from state file"}
-INFO:__main__:{"INFO": "Initializing tabpy"}
-INFO:__main__:{"INFO": "Done initializing tabpy"}
-INFO:__main__:{"INFO": "Web service listening on port 9004"}
-
-```
-
-At this point the server is ready to execute Python code remotely, or to deploy Python functions. See the [client documentation](client.md) for instructions on how to deploy Python functions.
-
-You can stop the server simply by terminating the python2.7 process which can be seen in Windows Task Manager. (Press Ctrl+Shift+Esc to access Task Manager)
-
-
-## Manual Installation
-
-If you are familiar with Python environments and have already set one up or prefer not to use Anaconda, you can skip the setup script, install just TabPy directly from the command line. 
-
-### Setting up an Anaconda Environment
-
-If you have Anaconda, it is optional but recommended you create a new Conda environment for this project. If you don't want to use Anaconda or create a new Conda environment for TabPy, you can directly jump to the [installation step](#installing-tabpy).
-
-```Batchfile
-
-conda create --name Tableau-Python-Server python=2.7 anaconda
-
-```
-The example above assumes Conda is defined as part of PATH and creates a Python 2.7 environment but Tableau Python Server is supported on both Python 2.7+ and Python 3.5+.
-
-Now activate the newly created environment.
-
-On Linux/MacOS:
-
-```bash
-
-source activate Tableau-Python-Server
-
-```
-
-On Windows:
-
-```Batchfile
-
-activate Tableau-Python-Server
-
-```
-### Installing TabPy
-
-Since TabPy is available on [PyPI](https://pypi.python.org/pypi/tabpy-server) you can install by simply running the following command.
-
-
-```Batchfile
-
-pip install tabpy-server
```

Comment:
I thought this was pretty confusing before as it muddles the process for using the source code vs a pre-built distribution. As you'll see in the changes I've created a separate Contributing guide for dealing with the source and simply updated the `server.md` file to note how to deal with pre-buil distributions 

Side note - that itself is admittedly kind of strange in the current iteration - would probably be better served with a CLI component than having to launch directly from site-packages